utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -68,6 +71,13 @@ class HttpFileStorage(
 
     private val httpClient by lazy {
         OkHttpClientHelper.buildClient {
+            val connectionPool = ConnectionPool(
+                HTTP_CLIENT_MAX_IDLE_CONNECTIONS,
+                HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS,
+                TimeUnit.SECONDS
```

Comment:
Sticking to the default of `TimeUnit.MINUTES` would make sense here to avoid the multiplication in `HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS` above.

Reply:
I prefer seconds as this allows a more fine grained adjustment. (In case the error should again pop up, I'd probably further reduce the value and then likely make it configurable). Apart from that the value is consistent with `HTTP_CLIENT_CONNECT_TIMEOUT_IN_SECONDS ` which is defined right next to it. I don't really see an issue with a multiplication.

Would you be ok with keeping this as-is?

Reply:
> The multiplication would get removed by the compiler anyway

This is not about performance, but about human readability.

> Would you be ok with keeping this as-is?

I think it's a bit odd, esp. as you otherwise say to repeat the default arguments, which strictly speaking you don't, as the `timeUnit` changes from `TimeUnit.MINUTES` to `TimeUnit.SECONDS`. But I believe it's not worth wasting more time by debating over it, so just keep it.

Reply:
> This is not about performance, but about human readability.

Ok, so I removed the multiplication to address this.

Reply:
The multiplication / `TimeUnit.SECONDS` is this there, but like I said, let's keep it like this then.